StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/Button+Extensions.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 extension Button.Configuration {
     static var financialConnectionsPrimary: Button.Configuration {
         var primaryButtonConfiguration = Button.Configuration.primary()
-        primaryButtonConfiguration.font = .stripeFont(forTextStyle: .bodyEmphasized)
+        primaryButtonConfiguration.font = FinancialConnectionsFont.label(.largeEmphasized).uiFont
```

Comment:
There are cases where we won't be able to set the line height. For example, this Stripe component only supports `UIFont`. This _should_ mostly be fine as these "extra" cases will hopefully only span 1 line (line height for 1 line doesn't matter in some cases...or at least cases where we already define how big the component will be [which we do for buttons]).